6tisch-roll-enrollment-priority.mkd

```diff
@@ -113,15 +115,15 @@ In the context of the {{!RFC6550}} RPL protocol, the term (2)"Join" has an alter
 This term "Join" has to do with preferred parent selection processes.
 
 In order to avoid the ambiguity of this term, this document refers to the process (1)"Join" as enrollment, leaving the term "Join" to mean (2)"Join".
-The term "onboarding" (or IoT Onboarding) is sometimes used to describe the enrollment process.
+The term "onboarding" (or IoT Onboarding) is increasingly used to describe what was called enrollment in other documents.
 However, the term *Join Proxy* is retained with it's meaning from {{!RFC9031}}.
 
 
 # Protocol Definition
 
 With this specification, the following option is defined for transmission in the DIO issued by the DODAG root and it MUST be propagated down the DODAG.
 
-A 6LR which would otherwise be willing to act as a *Join Proxy*, will examine the minimum priority field, and to that number, add any additional local consideration (such as upstream congestion).
+A 6LR which would otherwise be willing to act as a *Join Proxy*, will examine the minimum priority field, and to that number, add any additional local consideration (such as upstream congestion, number of NCE slots available, etc.)
 
 The Enrollment Priority can only be increased by each 6LR in value, to the maximum value of 0x7f.
```

Comment:
Are we allowing the 6LRs to change the Enrollment priority? Based on @mcr mail, it was not allowed.

Reply:
We aren't changing the global enrollment priority, but in translating this RPL metric to the one in RFC9032, the local node is allowed to take into consideration local conditions. Perhaps this isn't worded well.